plugins/inputs/statsd/statsd.go

```diff
@@ -214,6 +236,10 @@ const sampleConfig = `
   ## http://docs.datadoghq.com/guides/dogstatsd/
   parse_data_dog_tags = false
 
+  ## Parses events in the datadog statsd format
+  ## http://docs.datadoghq.com/guides/dogstatsd/
+  parse_data_dog_events = false
```

Comment:
Update for new option; don't include this link unless we have one that describes events/tags more precisely.